wss/1.0/modules/ROOT/pages/validate-wss-examples.adoc

```diff
@@ -0,0 +1,462 @@
+    <soap:Body><xenc:EncryptedData Id="ED-D77EFA434E6694DA5315531006937494" Type="http://www.w3.org/2001/04/xmlenc#Content" xmlns:xenc="http://www.w3.org/2001/04/xmlenc#"><xenc:EncryptionMethod Algorithm="http://www.w3.org/2001/04/xmlenc#aes128-cbc"/><ds:KeyInfo xmlns:ds="http://www.w3.org/2000/09/xmldsig#"><wsse:SecurityTokenReference wsse11:TokenType="http://docs.oasis-open.org/wss/oasis-wss-soap-message-security-1.1#EncryptedKey" xmlns:wsse="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd" xmlns:wsse11="http://docs.oasis-open.org/wss/oasis-wss-wssecurity-secext-1.1.xsd"><wsse:Reference URI="#EK-D77EFA434E6694DA5315531006937483"/></wsse:SecurityTokenReference></ds:KeyInfo><xenc:CipherData><xenc:CipherValue>vxJMblfqM9TtmZQmSWL9UzAVqSRWQSJMKxex4Lc8T0g3l/jPNaWQpUnyIdb8aMYSYbMSTfhpj4HlNnmiSm/rpboyBo/SkIRLdils/VE+JFjX4JkbRSefj4XDEY6g0hthQm5U4Z4B6NkdUQgd8xiumYW5PpqWHEsNEsZLE9udyPMS1VlEfZYMiMNhtVRPWkk5xBpH1LKx8P5k0UwgIxDUxOEDleJEt5CotalVH0FsffXHKI/lDcGHuY2HuTAkTW9UYkoascB9Ak3J5ypv4F/Nn1hmtJBsGSRd18auWDoXU7s=</xenc:CipherValue></xenc:CipherData></xenc:EncryptedData></soap:Body>
+</soap:Envelope>
+------=_Part_8049_2119795515.1555963425591
+Content-Type: application/json
+Content-Disposition: related; name="file1"; filename="a.json"
+Content-ID: file1
+
+{
+  "title": "Java 8 in Action",
+  "author": "Mario Fusco",
+  "year": 2014
+}
+------=_Part_8049_2119795515.1555963425591--
+----
+
+.Decryption Example
+[source,XML,linenums]
+----
+<wss:inbound-config name="decryption-config">
+  <wss:decryption-config>
+    <wss:keystore-config path="certificates/decrypt-keystore.jks"
+                            password="changeit"
+                            alias="s1as"
+                            keyPassword="changeit" />
+  </wss:decryption-config>
+</wss:inbound-config>
+
+<flow name="OrderTshirtServiceFlow">
+  <http:listener config-ref="HTTP-listener-config" path="/order" />
+  <wss:validate-wss config-ref="decryption-config" version="SOAP_12"/>
+  <flow-ref name="OrderTshirtFlowImpl" />
+</flow>
+----
+
+[[usernameToken]]
+== Validate UsernameToken
+
+The following examples show you how to validate against an <<ldap-validate,LDAP server>>, or a given <<username-password,username/password credentials pair>>.
```

Comment:
```suggestion
The following examples show you how to validate against either an <<ldap-validate,LDAP server>> or a provided <<username-password,username/password credentials pair>>.
```
I don't understand what comprise the pair: username and password + username and/or password?
We want to avoid using / to mean "and/or", but maybe this is syntax in this instance?